cmd/node/list_test.go

```diff
@@ -0,0 +1,198 @@
+package node
+
+import (
+	"bytes"
+	"testing"
+
+	relaypb "github.com/cilium/cilium/api/v1/relay"
+
+	"github.com/stretchr/testify/assert"
+
```

Comment:
nit: woud you mind removing these extra spaces?

Reply:
Yes, I did it here with squashing: https://github.com/cilium/hubble/pull/496/commits/1204b43f86c6f83492b02fc7719ef07408d0df54#diff-42e42a092b0a008f72b7e650d7264f339b132755c66ecbab050db7709c930146R3-R11